example/src/serde.rs

```diff
@@ -1,22 +1,43 @@
-use cel_interpreter::{Context, Program};
+use cel_interpreter::{Context, Duration, Program, Timestamp};
 use serde::Serialize;
 
 // An example struct that derives Serialize
 #[derive(Serialize)]
 struct MyStruct {
     a: i32,
     b: i32,
+    // To preserve durations and timestamps, use the cel_interpreter wrapper types.
+    c: Duration,
+    d: Timestamp,
```

Comment:
I feel like the changes in this file are valuable to have in the form of a test, but I feel that they muddy the simplicty of this example which was just intended to show how to pass a `T: Serialize` to the context. Could we move this into a test somewhere else, probably in the ser.rs or functions.rs file?

Reply:
I moved these to testable examples on the `Duration` and `Timestamp` types, does that work? The existing tests in `ser.rs` made this a bit redundant as normal tests